comp413-2017/RDFS
# WebRDFS server process, placeholder HTTP handlers

### Motivation

First step in implementing WebRDFS support. Future developers can build on this skeleton.

### Changeset

* Add `web-rdfs` directory and compile step
* Create `webrdfs` executable, starting HTTP server on a user-specified port
* Add dummy/placeholder handlers for all verbs on the single respected path per the API: https://hadoop.apache.org/docs/r1.0.4/webhdfs.html

### Testing

Manual test:

1. `./build/web-rdfs/webrdfs`
2. `curl localhost:8000/webhdfs/v1/path`
3. Observe HTTP 200 and closed connection

This PR depends on #146, #141.

This closes #129.
